src/main/resources/assets/energizedpower/lang/it_it.json

```diff
@@ -18,37 +18,37 @@
   "item.energizedpower.iron_plate": "Piastra di ferro",
   "item.energizedpower.gold_plate": "Piastra d'oro",
 
-  "item.energizedpower.iron_gear": "Iron Gear",
+  "item.energizedpower.iron_gear": "Ingranaggio di ferro",
 
-  "item.energizedpower.iron_rod": "Iron Rod",
+  "item.energizedpower.iron_rod": "Barra di ferro",
 
-  "item.energizedpower.copper_wire": "Copper Wire",
-  "item.energizedpower.gold_wire": "Gold Wire",
+  "item.energizedpower.copper_wire": "Cavo di rame",
```

Comment:
Is there a different translation for wire?
There are the Cables and Wires. Cables are blocks which transfer energy and wires are items which are used in crafting recipes only.

Reply:
Otherwise Copper Cable and Copper Wire would have the same name in the Italian translation.

Reply:
"Filo di rame"